lms/static/sass/views/_shoppingcart.scss

```diff
@@ -637,390 +532,480 @@
-          }
-          &:hover{background: $m-blue-d2;box-shadow: none;}
-         }
-        p{
-          font-family: "Open Sans",Verdana,Geneva,sans-serif,sans-serif;
-          padding: 13px 0;
-          text-align: right;
+        label {
+          @include padding($baseline*0.4, $baseline*0.75, $baseline*0.4, $baseline*0.3);
+          @extend %t-icon5;
+          @extend %t-regular;
+          display: inline-block;
+          margin-bottom: 0;
+          color: $dark-gray2;
+          font-family: $sans-serif;
+          font-style: normal;
         }
-        form{
-          position: relative;
+      }
+
+      input[type="radio"] {
+        @include margin-left($baseline/2);
+      }
+    }
+
+    .col-2 {
+      @include float(right);
+      width: 65%;
+
+      button[type="submit"] {
+        @include float(right);
+        @include margin-left($baseline*0.75);
+        @include padding($baseline*0.9, $baseline*3, $baseline*1.1, $baseline*1.5);
+        @extend %t-icon3;
+        @extend %t-strong;
+        width: auto;
+        height: ($baseline*3.35);
+        letter-spacing: 0;
+
+        // CASE: has id register
+        &#register {
+          padding: ($baseline*0.9) ($baseline*1.5);
         }
-          .fa-caret-right {
-            position: absolute;
-            right: 30px;
-            top: 22px;
-            color: white;
-            font-size: 24px;
-          }
 
-        label.pull-right{
-          font-family: "Open Sans",Verdana,Geneva,sans-serif,sans-serif;
-          font-style: normal;
-          text-align: right;
-          padding: 10px 25px 10px;
-          display: inline-block;
-          float: right;
-          line-height: 20px;
-          color: $dark-gray1;
+        // STATE: hover
+        &:hover {
+          box-shadow: none;
+          background: $m-blue-d2;
         }
       }
+
+      p {
+        @include text-align(right);
+        padding: ($baseline*0.65) 0;
+        font-family: $sans-serif;
+      }
+
+      form {
+        position: relative;
+      }
+
+      .fa-caret-right {
+        @include right($baseline*1.5);
+        @extend %t-icon3;
+        position: absolute;
+        top: ($baseline*1.1);
+        color: $white;
+      }
+
+      label.pull-right {
+        @include float(right);
+        @include text-align(right);
+        @include line-height(13.51);
+        display: inline-block;
+        padding: ($baseline/2) ($baseline*1.25);
+        color: $dark-gray1;
+        font-family: $sans-serif;
+        font-style: normal;
+      }
     }
-  .disclaimer{
-    color: $light-gray2;
+  }
+
+  .disclaimer {
+    @include text-align(right);
+    @extend %t-light;
     padding: ($baseline/2) 0;
-    text-align: right;
-    font-weight: 300;
+    color: $light-gray2;
   }
-  h3{
-    font-family: "Open Sans",Verdana,Geneva,sans-serif,sans-serif;
-    font-size: 16px;
-    font-weight: 400;
-    padding: 30px 20px;
+
+  h3 {
+    @extend %t-icon5;
```

Comment:
I won't comment on the others, but there are a number of other headings and text areas using the icon typography extends instead of the title or body copy extends. This would be a nice small thing to clear up. Thanks!